src/applications/disability-benefits/all-claims/pages/secondaryIncidentDescription.js

```diff
@@ -29,7 +29,7 @@ export const schema = index => ({
     [`secondaryIncident${index}`]: {
       type: 'object',
       properties: {
-        description,
+        incidentDescription: description,
```

Comment:
Should this property still be called [`description`](https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/dist/21-526EZ-ALLCLAIMS-schema.json#L1538)?

Reply:
Hmm...

`incidentDescription` is correct.  I updated the property on the `ptsdIncident` to be `incidentDescription` but missed the `ptsdSecondayincident` description property.

Reply:
I talked with @dmccurdybah "offline" (Slack) and he clarified: These `secondaryIncident${index}` properties get changed into items in the `form0781.incidents` array. These items do indeed have `incidentDescription`, not `description`.

Thanks for the clarification, @dmccurdybah! :raised_hands: